### ISSUE TITLE
Add JSONSchema tests

### ISSUE THREAD

**Issue by bjdmeest** [TiboStr/dataflow-visualization#102]
Just a thing that validates that the existing examples are in line with the generated JSONSchemas, and maybe a prehook that makes sure that the JSONSchemas are always updated and tested before committing. Depends on #99

**Comment by TiboStr** [comment]
@bjdmeest, is this a good tutorial to add a prehook https://dev.to/akdevcraft/git-pre-hook-pre-commit-hook-with-npm-project-44p2? This tutorial uses a bash script. But that means if someone wants to work on this project using Windows, that person can't commit?

**Comment by TiboStr** [comment]
Or should I open another issue that addresses this problem? So a Windows user (I don't have Windows on my machine) can solve that using (probably) a bat file.

**Comment by bjdmeest** [comment]
Or maybe just use smth like https://github.com/typicode/husky so we don't need to fix that ourselves?
